lib/contact_manager.rb

```diff
@@ -1,7 +1,7 @@
 class ContactManager
   require 'manage_phonebook'
```

Comment:
Usually `require` statements live at the top of the file but as you see this still works